src/TextField/TextField.tsx

```diff
@@ -94,149 +105,168 @@ export const TextField: React.FC<Props> = ({
   className,
   defaultValue,
   description,
+  inputAs = "input",
   disabled,
   error,
   helper,
   icon,
   label,
+  onBlur,
```

Comment:
Curious why this PR touches `onBlur` and what `otherProps` used to do?

Reply:
I tried to sneak this in because I thought it would go through fast. I'm going to pull it into another patch PR. I explicitly don't want to accept `otherProps` because, this being a layout component, it's not obvious where those props will be spread to.

Reply:
Now I remember why I did it this way; these two work together because there are formatting changes that will cause merge conflicts and I wanted to avoid that.